gene_transformer/blast.py

```diff
@@ -3,94 +3,87 @@
 import statistics
 import subprocess
 from pathlib import Path
-import pandas as pd
+import pandas as pd  # type: ignore[import]
+from typing import Optional, List
 
 
 class BlastRun:
     """Class to handle blasting single sequence against training sequence database"""
 
     def __init__(
         self,
-        sequence,
-        database_file,
-        temp_fasta_dir="/tmp/mzvyagin/",
-        temp_csv_dir="/tmp/mzvyagin/",
-    ):
-        self.sequence = sequence
+        sequence: str,
+        database_file: str,
+        temp_fasta_dir: Path,
+        temp_csv_dir: Path,
+    ) -> None:
         self.database_file = database_file
 
-        self.temp_fasta_dir = temp_fasta_dir
-        self.temp_fasta = Path(
-            str(temp_fasta_dir) + "/test_seq{}.fasta".format(hash(sequence))
-        )
-        # need to save sequence as fasta file in order to run Blast
+        self.temp_fasta = temp_fasta_dir / f"test_seq{hash(sequence)}.fasta"
+        # Need to save sequence as fasta file in order to run Blast
         make_fasta_from_seq(sequence, self.temp_fasta)
 
-        self.temp_csv_dir = temp_csv_dir
-        self.temp_csv = Path(
-            str(temp_csv_dir) + "/blast_res{}.csv".format(hash(sequence))
-        )
+        self.temp_csv = temp_csv_dir / f"blast_res{hash(sequence)}.csv"
 
-        # cannot get scores until blast has been run
+        # Cannot get scores until blast has been run
         self.ran_blast = False
 
-        self.scores = []
-        self.top_score = None
+        # TODO: Is this the correct type?
+        self.scores: List[float] = []
```

Comment:
@maxzvyagin does blast/the pandas csv, return a list of floats? Or is it a list of ints?